### PR TITLE
Mover as backend

### PR DESCRIPTION
**Pull request by johandahlberg**
A semi-large PR this time...

This introduces mover as the actual backend for the delivery service. It also introduces the possibility of skipping running mover (so that we can run integration tests without making real deliveries). 

Have a look and let me know what you think.

### CODE REVIEW

**Comment by johandahlberg** [comment]
I also added @b97pla as a review to this since he has the most experience working with mover so far.

File: delivery/services/delivery_service.py [hunk 1]
```diff
@@ -79,40 +89,54 @@ def deliver_by_staging_id(self, staging_id, delivery_project, md5sum_file):
...
+    @staticmethod
+    def _parse_status_from_mover_info_result(mover_info_result):
+        #Parse status from this type of example string:
+        # Delivered: Jan 19 00:23:31 [1484781811UTC]
+        pattern = re.compile('^(\w+):\s')
+        hits = pattern.match(mover_info_result)
+        if hits:
+            return hits.group(1)
+        else:
+            raise CannotParseMoverOutputException("Could not parse mover id from: {}".format(mover_info_result))
```
Comment (b97pla) [inline delivery/services/delivery_service.py]: Error message seems slightly off - should it be something like `"Could not parse mover status for delivery id {} from: {}"`

**Comment by b97pla** [comment]
Nice, I think it looks good :+1:

**Comment by johanherman** [comment]
:+1:

**Pull request merged**